index.html

```diff
@@ -2162,13 +2132,6 @@ <h3>Presentations</h3>
 <a>holder</a> is expected to either be obtained via the securing mechanism, or
 to not pertain to the <a>validation</a> of the <a>verifiable presentation</a>.
           </dd>
-          <dt><var>proof</var></dt>
-          <dd>
-The <a>verifiable presentation</a> MAY include a <code>proof</code>
-<a>property</a>. If present, the value SHOULD be used to express a securing
```

Comment:
normative statement point to non normative note.... and specific to data integrity.

Reply:
Split the sentence into two sentences, then. 

-1 for removing the statement.